.gitignore

```diff
@@ -14,8 +15,11 @@ out-editor-min/
 out-monaco-editor-core/
 out-vscode/
 out-vscode-min/
-build/node_modules
+**/node_modules
 coverage/
 test_data/
 test-results/
 yarn-error.log
+**/*.json.bak
+**/*.ts.bak
```

Comment:
Why are these being added? I wouldn't expect these to normally exist so let's not clutter up the file by adding more stuff

Reply:
These are from a tool that I use in my development environment called winmerge. Removing these